model/encodable/keys.go

```diff
@@ -38,6 +39,14 @@ func fromMsgPackHex(b []byte) ([]byte, error) {
 	return hex.DecodeString(x)
 }
 
+func fromCBORPackHex(b []byte) ([]byte, error) {
+	var x string
+	if err := cbor.Unmarshal(b, &x); err != nil {
+		return nil, err
```

Comment:
@yhassanzadeh13 thanks for the comment! Just pushed the update :-)

FYI this happened because I just copy and pasted the code from `fromMsgpackPackHex()` which was probably copy and pasted from `fromJSONPackHex()` too. I updated their error messages too! :-)